uniffi_bindgen/src/scaffolding/templates/TopLevelFunctionTemplate.rs

```diff
@@ -14,3 +15,24 @@ pub extern "C" fn {{ func.ffi_func().name() }}(
     uniffi::deps::log::debug!("{{ func.ffi_func().name() }}");
     {% call rs::to_rs_function_call(func) %}
 }
+{% endfor -%}
+
+// Create a macro that can be used by a dependent create to forward all of the scaffolding
+// functions defined here.  This is used in 2 ways:
+//
+//   - Building libraries that contain multiple UniFFI components (megazord, libxul).  The combined
+//     library can depend on each component then call uniffi_forward_scaffolding!() for each one.
+//   - Test fixtures.  Each binding generator needs to test against the test fixtures.  To allow
+//     this, they call uniffi_forward_scaffolding!() for the test fixture in their test module.
+#[macro_export]
+macro_rules! uniffi_forward_scaffolding {
```

Comment:
Hmm I'm thinking about non-high level function forwarding (like forwarding to a method on an object) - how would that work? or would forwarding just the high-level functions be enough for our fixtures?

Reply:
Great point.  I thought I was getting those too, but I wasn't.  Just pushed a new commit that I think should do this properly.